Yosemite/Yosemite/Stores/CardPresentPaymentStore.swift

```diff
@@ -604,3 +666,49 @@ public protocol CardReaderCapableRemote {
 
 extension WCPayRemote: CardReaderCapableRemote {}
 extension StripeRemote: CardReaderCapableRemote {}
+
+// MARK: - WCPayChargesError
+public enum WCPayChargesError: Error, LocalizedError {
+    case noSuchChargeError(message: String)
+    case otherError(error: AnyError)
+
+    init(underlyingError error: Error) {
+        guard case let DotcomError.unknown(code, message) = error,
+        let message = message else {
+            self = .otherError(error: error.toAnyError)
+            return
+        }
```

Comment:
The indentation here isn't right. Kind of a nitpick, but it could lead to confusion if one's not paying close attention.

```suggestion
        guard case let DotcomError.unknown(code, message) = error,
              let message = message else {
                  self = .otherError(error: error.toAnyError)
                  return
              }
```